projects/plugins/jetpack/changelog/display-map-block-as-preview-if-in-block-preview-mode

```diff
@@ -0,0 +1,4 @@
+Significance: patch
+Type: other
+
+Map block: Display a static map image when the block is rendered in a pattern preview
```

Comment:
Feels like we also need a release note about the change to the block attribute now

Reply:
I'm unsure since it's an internal attribute.

Reply:
Since attributes are written to post content they don't seem internal. We need to stay compatible with the existing data in folks' databases. The chances that someone would have post content that includes `preview:true` is pretty low though, since it's not something users can control in the block settings. And it's also quite hard to do manually using the code editor. Maybe it's a 0% chance someone would have a block with this attribute in the database? It's kind of hard to tell.

Maybe someone manually created a block pattern using a text editor and they used the `preview` attribute? I'm not sure how far fetched this is.

Reply:
Fair. Let's make them aware.